.mailmap

```diff
@@ -1,62 +1,176 @@
+
 Cimarron Mittelsteadt <cimarronm@gmail.com> Cimarron <cimarronm@gmail.com>
+
+DanHickstein <danhickstein@gmail.com>
+
+Daniel Hyams <dhyams@gmail.com>
 Daniel Hyams <dhyams@gmail.com> Daniel Hyams <dhyams@gitdev.(none)>
-Daniel Hyams <dhyams@gmail.com> dhyams <dhyams@gmail.com>
-Eric Dill <thedizzle@gmail.com> Eric Dill <edill@bnl.gov>
-Erik Bray <erik.m.bray@gmail.com> Erik M. Bray <embray@stsci.edu>
+
+David Kua <david@kua.io> <david.kua@mail.utoronto.ca>
+
+Devashish Deshpande <ashu.9412@gmail.com>
+
+endolith <endolith@gmail.com>
+
+Eric Dill <thedizzle@gmail.com> <edill@bnl.gov>
+
+Erik Bray <erik.m.bray@gmail.com> <embray@stsci.edu>
+
+Eric Ma <ericmajinglong@gmail.com> <ericmjl@Erics-MacBook-Pro.local>
+Eric Ma <ericmajinglong@gmail.com> <ericmjl@users.noreply.github.com>
+
+Filipe <ocefpaf@gmail.com>
+
+Florian LB <florian.s.lebourdais@gmail.com>
+
 Francesco Montesano <franz.bergesund@gmail.com> montefra <franz.bergesund@gmail.com>
-Jake Vanderplas <jakevdp@gmail.com> Jake Vanderplas <jakevdp@yahoo.com>
-Jake Vanderplas <jakevdp@gmail.com> Jake Vanderplas <vanderplas@astro.washington.edu>
-James R. Evans <jrevans1@earthlink.net> James Evans <jrevans1@earthlink.net>
-Jeffrey Bingham <bingjeff@gmail.com> Jeff Bingham <bingjeff@gmail.com>
-Jens Hedegaard Nielsen <jenshnielsen@gmail.com> Jens Hedegaard Nielsen <jens.nielsen@ucl.ac.uk>
-Jens Hedegaard Nielsen <jenshnielsen@gmail.com> Jens H Nielsen <jenshnielsen@gmail.com>
-Jens Hedegaard Nielsen <jenshnielsen@gmail.com> Jens H. Nielsen <jenshnielsen@gmail.com>
-Jens Hedegaard Nielsen <jenshnielsen@gmail.com> Jens H. Nielsen <jens.nielsen@ucl.ac.uk>
-John Hunter <jdh2358@gmail.com> jdh2358 <jdh2358@gmail.com>
-Jouni K. Seppänen <jks@iki.fi> Jouni K. Seppänen <jks@iki.fi>
-Julien Schueller <julien.schueller@gmail.com> jschueller <schueller@porsche-l64.phimeca.lan>
-Julien Schueller <julien.schueller@gmail.com> Julien Schueller <schueller@bx-l64.phimeca.lan>
-Julien Schueller <julien.schueller@gmail.com> Julien Schueller <schueller@porsche-l64.phimeca.lan>
-Kevin Davies <kdavies4@gmail.com> Kevin Davies <daviesk24@yahoo.com>
-kikocorreoso <kikocorreoso@gmail.com> kikocorreoso <kikocorreoso@users.noreply.github.com>
-Lennart Fricke <lennart@die-frickes.eu> Lennart Fricke <lennart.fricke@kabelmail.de>
-Levi Kilcher <levi.kilcher@nrel.gov> Levi Kilcher <Levi.Kilcher@nrel.gov>
-Martin Fitzpatrick <martin.fitzpatrick@gmail.com> Martin Fitzpatrick <mfitzp@abl.es>
-Matthias Bussonnier <bussonniermatthias@gmail.com> Bussonnier Matthias <bussonniermatthias@gmail.com>
-Matthias Bussonnier <bussonniermatthias@gmail.com> Matthias BUSSONNIER <bussonniermatthias@gmail.com>
-Michael Droettboom <mdboom@gmail.com> Michael Droettboom <mdroe@stsci.edu>
+
+hannah <story645@gmail.com>
+
+Hans Moritz Günther <moritz.guenther@gmx.de>
+
+J. Goutin <JGoutin@users.noreply.github.com> JGoutin <ginnungagap@free.fr>
+
+Jack Kelly <jack-list@xlk.org.uk> Jack (aka Daniel) Kelly <daniel.kelly10@imperial.ac.uk>
```

Comment:
Please could we change my line to:

`Jack Kelly <jack.kelly@imperial.ac.uk>`

Thank you!